src/hotspot/share/jfr/utilities/jfrAllocation.hpp

```diff
@@ -53,9 +53,9 @@ class JfrCHeapObj : public CHeapObj<mtTracing> {
   static char* allocate_array_noinline(size_t elements, size_t element_size);
 
  public:
-  NOINLINE void* operator new(size_t size) throw();
+  NOINLINE void* operator new(size_t size);
   NOINLINE void* operator new (size_t size, const std::nothrow_t&  nothrow_constant) throw();
-  NOINLINE void* operator new [](size_t size) throw();
+  NOINLINE void* operator new [](size_t size);
```

Comment:
The changes to JfrCHeapObj are not correct, because these allocators currently _can_ return null.
Their implementation is just to return the result of calling the non-throwing allocator.  That's probably
not an ideal implementation.  Either the declaration needs to be left as-is or the implementation changed.

Reply:
declaration kept as it is.